src/cdk-experimental/menu/menu-item.spec.ts

```diff
@@ -5,58 +5,88 @@ import {CdkMenuModule} from './menu-module';
 import {CdkMenuItem} from './menu-item';
 
 describe('MenuItem', () => {
-  let fixture: ComponentFixture<SingleMenuItem>;
-  let button: CdkMenuItem;
-  let nativeButton: HTMLButtonElement;
-
-  beforeEach(async(() => {
-    TestBed.configureTestingModule({
-      imports: [CdkMenuModule],
-      declarations: [SingleMenuItem],
-    }).compileComponents();
-  }));
-
-  beforeEach(() => {
-    fixture = TestBed.createComponent(SingleMenuItem);
-    fixture.detectChanges();
-
-    button = fixture.debugElement.query(By.directive(CdkMenuItem)).injector.get(CdkMenuItem);
-    nativeButton = fixture.debugElement.query(By.directive(CdkMenuItem)).nativeElement;
-  });
+  describe('without enclosing elements', () => {
+    let fixture: ComponentFixture<SingleMenuItem>;
+    let menuItem: CdkMenuItem;
+    let nativeButton: HTMLButtonElement;
 
-  it('should have the menuitem role', () => {
-    expect(nativeButton.getAttribute('role')).toBe('menuitem');
-  });
+    beforeEach(async(() => {
+      TestBed.configureTestingModule({
+        imports: [CdkMenuModule],
+        declarations: [SingleMenuItem],
+      }).compileComponents();
+    }));
 
-  it('should coerce the disabled property', () => {
-    (button as any).disabled = '';
-    expect(button.disabled).toBeTrue();
-  });
+    beforeEach(() => {
+      fixture = TestBed.createComponent(SingleMenuItem);
+      fixture.detectChanges();
 
-  it('should toggle the aria disabled attribute', () => {
-    expect(nativeButton.getAttribute('aria-disabled')).toBeNull();
+      menuItem = fixture.debugElement.query(By.directive(CdkMenuItem)).injector.get(CdkMenuItem);
+      nativeButton = fixture.debugElement.query(By.directive(CdkMenuItem)).nativeElement;
+    });
 
-    button.disabled = true;
-    fixture.detectChanges();
+    it('should have the menuitem role', () => {
+      expect(nativeButton.getAttribute('role')).toBe('menuitem');
+    });
 
-    expect(nativeButton.getAttribute('aria-disabled')).toBe('true');
+    it('should coerce the disabled property', () => {
+      (menuItem as any).disabled = '';
+      expect(menuItem.disabled).toBeTrue();
+    });
 
-    button.disabled = false;
-    fixture.detectChanges();
+    it('should toggle the aria disabled attribute', () => {
+      expect(nativeButton.getAttribute('aria-disabled')).toBeNull();
```

Comment:
I recommend using `hasAttribute` to check for the absence of an attribute rather than checking that the value is null